Map Tools - Show GPS on Map when equipped with UAV Terminal (#9885)
acemod/ACE3: docs/wiki/feature/maptools.md

This adds a plotting board that can be used to aid in the rapid usage and adjustment of short-ranged indirect fires, as well as quick measurements of directions and distances between points, and general land-navigation.

### 1.4 GPS on map

> If you are equipped with a vanilla GPS or UAV Terminal, a GPS-display will be shown on the map, indicating current grid position. (You don't need the `Map Tools` item in your inventory for this.)

Reviewer: ```suggestion
If you are equipped with a vanilla GPS or UAV Terminal, a GPS display will be shown on the map, indicating current grid position. (You don't need the `Map Tools` item in your inventory for this.)
```

> If you are equipped with a vanilla GPS or UAV Terminal, a GPS display will be shown on the map, indicating current grid position. (You don't need the `Map Tools` item in your inventory for this.)

Reviewer 2: ```suggestion
If you have GPS capability, a GPS display will be shown on the map, indicating current grid position. (You don't need the `Map Tools` item in your inventory for this.)
```
